tests/baselines/reference/project/realisticNpmTypes/amd/realisticNpmTypes.json

```diff
@@ -0,0 +1,19 @@
+{
+    "scenario": "Realistic npm package types",
+    "projectRoot": "tests/cases/projects/realisticNpmTypes",
+    "inputFiles": [
+        "index.ts"
+    ],
+    "declaration": true,
+    "baselineCheck": true,
+    "moduleResolution": "node",
+    "resolvedInputFiles": [
+        "lib.d.ts",
+        "node_modules/@types/m1/index.d.ts",
```

Comment:
Hah. It didn;t get committed because it's in a node_modules folder. That's funny. And annoying. Let me fix that.